packages/components/src/interfaces/internal/Notification.ts

```diff
@@ -2,26 +2,25 @@ import type { Static } from "@sinclair/typebox"
 import { Type } from "@sinclair/typebox"
 
 import type { IsomerSiteProps, LinkComponentType } from "~/types"
-import { TextSchema } from "../native/Text"
+import { SimpleProseSchema } from "../native/Prose"
 
 export const NotificationSchema = Type.Object(
   {
-    title: Type.Optional(
-      Type.String({
-        title: "Notification title",
-        description: "The title of the notification",
-        format: "hidden",
-      }),
-    ),
-    content: Type.Array(TextSchema, {
-      title: "Notification content",
-      description: "The content of the notification",
-      format: "hidden",
+    enabled: Type.Boolean({
```

Comment:
question: hmm what do you think about doing an optional object instead, so something like this:

```ts
{
    notification: Type.Optional(Type.Object(
        {
            title: Type.String({...}),
            content: SimpleProseSchema
        }, {
            title: "Display a banner",
            description: "The site notification will always be visible on the site until it is dismissed by the user.",
        }
    ))
}
```

So the existence of the `notification` object determines if it's enabled or not.